h_pyramid_sentry/__init__.py

```diff
@@ -39,6 +39,7 @@ def includeme(config):
     ]
 
     if asbool(config.registry.settings.get("h_pyramid_sentry.retry_support")):
+        # pylint:disable=import-outside-toplevel
```

Comment:
```suggestion
        # pylint:disable=import-outside-toplevel
        # This is here to lazy load only when required
```
Might be nice to explain why this isn't in the usual place